### PR TITLE
EOS-14981: Fill ds fsal_dynamicfsinfo_t to zeroes and than pass to nfs

### PR DESCRIPTION
**Pull request by gshipra**
Problem Statement:
[Testing.txt](https://github.com/Seagate/cortx-fs-ganesha/files/5514235/Testing.txt)

EOS-14981: Fill ds fsal_dynamicfsinfo_t to zeroes and than pass to nfs
Problem Description:
We do not have fs stats implemented yet, get_dynamic_info() should
return
zeroed out data structure fsal_dynamicfsinfo_t.
Solution Overview:
Zeroed out fsal_dynamicfsinfo_t in get_dynamic_info()
Unit Test Cases:
1.UT passes
2.df -h /mnt/xxc shows 0.
Signed-off-by: Shipra <shipra.gupta@seagate.com>

### CODE REVIEW

**Comment by gshipra** [comment]
UT outout:

Configuration Completed
Clean indexes prepared
NSAL Unit tests
NS Tests
Test results are logged to /var/log/cortx/test/ut/ut_nsal.log
Total tests  = 5
Tests passed = 5
Tests failed = 0

Iterator test
Test results are logged to /var/log/cortx/test/ut/ut_nsal.log
Total tests  = 2
Tests passed = 2
Tests failed = 0

KVTree test
Test results are logged to /var/log/cortx/test/ut/ut_nsal.log
Total tests  = 17
Tests passed = 17
Tests failed = 0

Global KVS Tests
Test results are logged to /var/log/cortx/test/ut/ut_nsal.log
Total tests  = 4
Tests passed = 4
Tests failed = 0

CORTXFS Unit tests
Endpoint ops Tests
Test results are logged to /var/log/cortx/test/ut/ut_cortxfs.logs
Total tests  = 4
Tests passed = 4
Tests failed = 0

FS Tests
Test results are logged to /var/log/cortx/test/ut/ut_cortxfs.log
Total tests  = 3
Tests passed = 3
Tests failed = 0

Directory tests
Test results are logged to /var/log/cortx/test/ut/ut_cortxfs.log
Total tests  = 14
Tests passed = 14
Tests failed = 0

File creation tests
Test results are logged to /var/log/cortx/test/ut/ut_cortxfs.log
Total tests  = 4
Tests passed = 4
Tests failed = 0

Link Tests
Test results are logged to /var/log/cortx/test/ut/ut_cortxfs.log
Total tests  = 6
Tests passed = 6
Tests failed = 0

Rename tests
Test results are logged to /var/log/cortx/test/ut/ut_cortxfs.log
Total tests  = 3
Tests passed = 3
Tests failed = 0

Attribute Tests
Test results are logged to /var/log/cortx/test/ut/ut_cortxfs.log
Total tests  = 5
Tests passed = 5
Tests failed = 0

Xattr file Tests
Test results are logged to /var/log/cortx/test/ut/xattr_file_ops.log
Total tests  = 9
Tests passed = 9
Tests failed = 0

Xattr dir Tests
Test results are logged to /var/log/cortx/test/ut/xattr_dir_ops.log
Total tests  = 9
Tests passed = 9
Tests failed = 0

IO tests
Test results are logged to /var/log/cortx/test/ut/ut_cortxfs.log
Total tests  = 7
Tests passed = 7
Tests failed = 0

DSAL Unit tests
Dsal basic test
Test results are logged to /var/log/cortx/test/ut/ut_dsal.log
Total tests  = 11
Tests passed = 11
Tests failed = 0

Dsal IO test
Test results are logged to /var/log/cortx/test/ut/ut_dsal.log
Total tests  = 2
Tests passed = 2
Tests failed = 0

Dsal space stats test
Test results are logged to /var/log/cortx/test/ut/ut_dsal.log
Total tests  = 2
Tests passed = 2
Tests failed = 0

df -h output WITHOUT fix:
-bash-4.2$ df -h /mnt/dir1
Filesystem          Size  Used Avail Use% Mounted on
127.0.0.1:/cortxfs  128T  242M     0 100% /mnt/dir1

df -h output WITH fix:
-bash-4.2$ df -h /mnt/dir1
Filesystem          Size  Used Avail Use% Mounted on
127.0.0.1:/cortxfs     0     0     0    - /mnt/dir1

File: src/FSAL/FSAL_CORTXFS/export.c [hunk 1]
```diff
@@ -231,6 +231,7 @@ static fsal_status_t get_dynamic_info(struct fsal_export *exp_hdl,
 				      struct fsal_obj_handle *obj_hdl,
 				      fsal_dynamicfsinfo_t *infop)
 {
+	memset(infop, 0, sizeof(fsal_dynamicfsinfo_t));
```
Comment (sachinpunadikar) [inline src/FSAL/FSAL_CORTXFS/export.c:234]: Please add a comment stating that this temporary arrangement for returning all 0s. Also mention that actual calculation of stats is needed & should be returned back.
Fix is correct.

**Review by sachinpunadikar — CHANGES_REQUESTED** [review]
Please add comment about the change

File: src/FSAL/FSAL_CORTXFS/export.c (hunk 1 above)
Comment (gshipra) [reply src/FSAL/FSAL_CORTXFS/export.c:239]: Added.

**Review by sachinpunadikar — APPROVED** [review]
+1

**Review by s-arya — APPROVED** [review]
LGTM. +1

**Pull request merged**